src/aap_eda/tasks/orchestrator.py

```diff
@@ -191,79 +209,96 @@ def dispatch(
         ):
             if not queue_name:
                 LOGGER.info(
-                    "Scheduling request"
-                    f"{request_type_text} for {process_parent_type}"
-                    f" {process_parent_id} to the least busy queue"
-                    "; it is not currently associated with a queue.",
+                    f"Scheduling {process_parent_type} {process_parent_id} "
+                    "to the least busy queue; it is not currently associated "
+                    " with a queue.",
```

Comment:
I don't think simply dropping the rewording, in this and other messages, is a necessity.  Yes, 863 eliminated the need for the `request_type_text` but including the request type in the message is helpful.  The message changes in 892, in light of 863, should be considered.

Reply:
I did not rewording, the differences are because I didn't work from the existing code but from the a reverted commit, so the messages are from the previous commit. I was more focused in the logic itself.  I can recover the messages as they are, no problem to me.

Reply:
messages updated, please review.